man/coredump.conf.xml

```diff
@@ -110,14 +110,14 @@
       </varlistentry>
 
       <varlistentry>
-        <term><varname>AccessContainer=</varname></term>
+        <term><varname>EnterNamespace=</varname></term>
 
         <listitem><para>Controls whether <command>systemd-coredump</command> will attempt to use the mount tree of
-        a process that crashed within a container. Access to the container's filesystem might be necessary to generate
+        a process that crashed in PID namespace. Access to the namespace's mount tree might be necessary to generate
```

Comment:
hmm, "PID namespace"?

your changes don't care about PID namespace, do they? they care about file system namespaces only afaics.

Reply:
I mention PID namespace because this is what implementation checks for. We attempt coredump forwarding or gathering mount tree when crashed process ran in a different namespace than our own PID namespace.

https://github.com/systemd/systemd/blob/main/src/coredump/coredump.c#L1729